lang/en.json

```
@@ -1137,6 +1138,17 @@
 "DND5E.PolymorphWildShape": "Wild Shape",
 "DND5E.Portrait": "Portrait",
 "DND5E.Prepared": "Prepared",
+"DND5E.Prerequisites": {
+  "Header": "Feature Prerequisites",
+  "FIELDS": {
+    "prerequisites": {
+      "level": {
+        "label": "Required Level",
+        "hint": "Character or class level required to select this feature as part of a choice advancement."
```

I'm not sure how meaningful 'choice advancement' is to a non-power-user DM who is not manually configuring advancements themselves. I know we don't want to run the risk of this being misunderstood as preventing players from adding these feats to their sheet via plain drag-and-drop, so maybe something like:

```suggestion
        "hint": "Character or class level required to select this feature when levelling up."
```